### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MishaLehotskyi**
- [DEMO LINK](https://MishaLehotskyi.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://MishaLehotskyi.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,40 @@
...
+    <header class="header">
+      <a class="logo" href="#home">
+        <img class="logo__img"
+             src="./images/logo.png"
+             alt="Moyo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active">Apple</a>
```
Comment (xiimiik) [inline src/index.html:21]: The link with class 'is-active' doesn't have an href attribute. All links should have a valid href attribute.

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,62 @@
...
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  position: relative;
```
Comment (xiimiik) [inline src/style.css:56]: The position: relative style on the .is-active::after pseudo-element does not have any effect as it does not have top, right, bottom, or left properties defined. If you want this element to be positioned relative to the .is-active element, you should make the .is-active element position: relative.

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,62 @@
...
+  line-height: 60px;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.nav__link:hover {
+  cursor: pointer;
```
Comment (xiimiik) [inline src/style.css:45]: The cursor: pointer style is unnecessary for .nav__link:hover as it's the default for anchor tags.

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,62 @@
...
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
```
Comment (xiimiik) [inline src/style.css:50]: Color property for .is-active class is missing, it should have a color property for normal state as well.

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,62 @@
...
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
```
Comment (xiimiik) [inline src/style.css:33]: Avoid using fixed height for the link. It's better to control the height with padding to handle the content overflow gracefully.

File: src/index.html [hunk 6]
```diff
@@ -9,6 +9,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#home">
+        <img class="logo__img"
+             src="./images/logo.png"
+             alt="Moyo">
```
Comment (xiimiik) [inline src/index.html:16]: If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.
```suggestion
        <img 
          class="logo__img"
          src="./images/logo.png"
          alt="Moyo"
        >
```

File: src/index.html [hunk 7]
```diff
@@ -9,6 +9,40 @@
...
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">Video</a>
+          </li>
```
Comment (xiimiik) [inline src/index.html:43]: Code style
```suggestion
          <li class="nav__item">
            <a class="nav__link is-active">Apple</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link">Samsung</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link">Smartphones</a>
          </li>
          
          <li class="nav__item">
            <a data-qa="hover" class="nav__link">Laptops & Computers</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link">Gadgets</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link">Tablets</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link">Photo</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link">Video</a>
          </li>
```

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,62 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
 body {
   margin: 0;
 }
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  flex-direction: row;
```
Comment (xiimiik) [inline src/style.css:10]: its a default value for flexbox

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,62 @@
...
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+  flex-direction: row;
```
Comment (xiimiik) [inline src/style.css:26]: default value

**Review by xiimiik — CHANGES_REQUESTED** [review]
GJ, left a few comments for getting ur code better!

File: src/style.css (hunk 4 above)
Comment (MishaLehotskyi) [reply src/style.css:49]: What do you mean by "Color property for .is-active class is missing"? I have to store color value in the variable?

File: src/style.css (hunk 5 above)
Comment (MishaLehotskyi) [reply src/style.css:31]: I set padding to 23px 0. I don't have other solutions(

File: src/index.html [hunk 10]
```diff
@@ -9,6 +9,52 @@
...
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#home">
+        <img
+          class="logo__img"
+          src="./images/logo.png"
+          alt="Moyo"
+        >
+      </a>
+      <nav class="nav">
```
Comment (DarkMistyRoom) [inline src/index.html:20]: Add empty lines between multiline sibling blocks of HTML

File: src/index.html [hunk 11]
```diff
@@ -9,6 +9,52 @@
...
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#laptops-and-computers"
+               data-qa="hover"
+               class="nav__link">Laptops & Computers
+            </a>
```
Comment (DarkMistyRoom) [inline src/index.html:38]:
```suggestion
            <a 
              href="#laptops-and-computers"
              data-qa="hover"
              class="nav__link"
            >
              Laptops & Computers
            </a>
```

File: src/style.css (hunk 5 above)
Comment (DarkMistyRoom) [reply src/style.css:36]: it's ok

**Review by DarkMistyRoom — APPROVED** [review]
Good job! But pay attention to my comments

**Pull request closed without merging**